### PR TITLE
The weather app

### PR DESCRIPTION
**Pull request by Lovisaaberg**
## Netlify link
https://lovisas-weather-app.netlify.app

## Collaborators
-

### CODE REVIEW

**Comment by fridaforser** [comment]
Hi Lovisa,

I think your code looks great! It was super easy to follow along and understand the flow in your app, both with your descriptive comments and also with how dry your code is. 
I especially liked how you added an else function for the other types of weathers and added a dedicated text for those, well done!

The only thing I have to comment on right now, and I might actually be wrong on this one since I checked the app in the afternoon, but if the time is before 12 when you check the app, do you see today in the forecast as well? I might just have an untrained eye, but it looks as if maybe it'll say Thursday as the first day in the forecast tomorrow morning as well? I might just be iffy since this was the part I struggled with the most myself, and just want to see if I overcomplicated things in my own code.

Other than the tiny comment about the forecast, everything looks awesome and the site looks exactly like the design template. Good job! 👏🏻

File: index.html [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  <link rel="stylesheet" href="./style.css" />
+</head>
+
+<body>
+<div id="weather-container"></div>
+<div id="city-container"></div>
+<div id="weather-week-container"></div>
+
+<script src="./script.js"></script>
+</body>
```
Comment (HIPPIEKICK) [inline index.html:17]: Don't forget indentation:
```
<body>
  <div id="weather-container"></div>
  <div id="city-container"></div>
  <div id="weather-week-container"></div>

  <script src="./script.js"></script>
</body>
```

File: script.js [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+        minute: "2-digit",
+      });
+
+      weatherContainer.innerHTML = `
+    ${json.weather[0].description} | ${json.main.temp.toFixed(1)}° 
+    <p>Sunrise ${timeForSunrise}</p><p>Sunset ${timeForSunset}</p>
+    `;
+
+      //Function to change backround color and icon depending on the weather.
+      const warmColdBackround = () => {
```
Comment (HIPPIEKICK) [inline script.js:33]: Maybe this function could be named changeBackground?

File: script.js [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+        if (json.weather[0].main === "Clouds") {
```
Comment (HIPPIEKICK) [inline script.js:34]: Maybe `json.weather[0].main` could be a variable?

**Review by HIPPIEKICK — APPROVED** [review]
Good job implementing the design! Your JavaScript looks good too, but I would love to see you creating more functions in the future even though this works as intended.

**Pull request closed without merging**